### PR TITLE
hooks: create locale configuration 

### PR DESCRIPTION
**Pull request by alfonsosanchezbeato**
This is read by pam and others.

### CODE REVIEW

**Review by Meulengracht — APPROVED** [review]

**Comment by alfonsosanchezbeato** [comment]
Please don't merge this yet as I do not see that the files are actually appearing in the snap.

**Comment by alfonsosanchezbeato** [comment]
Removing blocked label after changing the approach

File: hooks/009-locale-archive.chroot [hunk 1]
```diff
@@ -23,4 +23,12 @@ for encoding in "${!found_encodings[@]}"; do
...
 done
 
+# Backup configuration
+cp /etc/locale.conf /etc/locale.conf.bak
+
+# purge
 apt purge -y locales
+
+# and restore so the default locale can be found
+mv /etc/locale.conf.bak /etc/locale.conf
```
Comment (valentindavid) [inline hooks/009-locale-archive.chroot:33]: Could we just put LANG=C.UTF-8, without taking it from the image=

Comment (valentindavid) [reply hooks/009-locale-archive.chroot:33]: Also it needs to be writable in theory. So we should add it to the writable-paths

Comment (alfonsosanchezbeato) [reply hooks/009-locale-archive.chroot]: I've changed this now

**Review by valentindavid — APPROVED** [review]
Thank you.

**Pull request merged**